gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManagerController.java

```diff
@@ -87,4 +94,24 @@ private ResponseEntity<byte[]> getZippedGroup(OffsetDateTime timestamp, String f
     public ResponseEntity<List<TaskDto>> getListTasksFromBusinessDate(@PathVariable String businessDate) {
         return ResponseEntity.ok().body(builder.getListTasksDto(LocalDate.parse(businessDate)));
     }
+
+    @GetMapping(value = "/file/{fileType}/{timestamp}", produces = "application/octet-stream")
```

Comment:
I propose to put /tasks/{timestamp}/.... to be in harmony with the other urls.

Reply:
OK then the URI will be /tasks/{timestamp}/file/{fileType}
this correction need to modify an other repo